content/applications/websites/website/web_design/visibility.rst

```diff
@@ -0,0 +1,53 @@
+
+.. |mobile icon| replace:: :raw-html:`<svg viewBox="0 0 566.93 566.93" xmlns="http://www.w3.org/2000/svg" width="20" height="20"><rect transform="translate(283.46 -117.41) rotate(45)" x="255.56" y="-16.93" width="55.81" height="600.8" rx="25.61"></rect><path d="m395.46 399.46a12 12 0 0 1-12 12h-128.4l-115.68 115.69a47.8 47.8 0 0 0 32.08 12.31h224a48 48 0 0 0 48-48v-268.4l-48 48zm-112 108a32 32 0 1 1 32-32 32 32 0 0 1-32 32z"></path><path d="m171.46 87.46a12 12 0 0 1 12-12h200a11.89 11.89 0 0 1 6.48 1.93l37.61-37.61a47.82 47.82 0 0 0-32.09-12.32h-224a48 48 0 0 0-48 48v268.41l48-48z"></path></svg>`
+
+To toggle the visibility of a building block based on the visitor's device type, open the website
+editor, and select a block or a column element within a block. In the :guilabel:`Customize` tab,
+scroll down to :guilabel:`Visibility` under the block or column customization options, and click the
+|mobile icon| (:guilabel:`Show/Hide on Mobile`) or |desktop icon| (:guilabel:`Show/Hide on Desktop`)
+button. Click on :guilabel:`Save` to apply the changes to your website.
+
+Conditions
+==========
+
+To access the language, country, and :doc:`UTM <../reporting/link_tracker>` conditions, open the
+website editor and select a building block. In the :guilabel:`Customize` tab, scroll down to
+:guilabel:`Visibility`, click :guilabel:`No condition`, and select :guilabel:`Conditionally` instead
+to display them.
+
+- :guilabel:`Country`: the country of the visitor's IP address.
+- :guilabel:`Languages`: the website language the visitor selected. This option is only available if
+  your website is translated into at least one language besides the original.
+- :guilabel:`UTM Campaign`
+- :guilabel:`UTM Medium`
+- :guilabel:`UTM Source`
+- :guilabel:`Users`: based on whether visitors are logged in or everyone.
+
+Click on :guilabel:`Save` to apply the changes to your website.
+
+Invisible elements
+==================
+
+Hidden blocks and elements are listed at the bottom of the :guilabel:`Blocks`, :guilabel:`Customize`
+, and :guilabel:`Edit` menus. Select one to display it in the website editor. This allows you to see
+the elements you have made invisible while working in the website editor, but it does not change
+their visibility status on your website's front-end. Click on :guilabel:`Save` to apply the changes
+to your website.
```

Comment:
```suggestion
Blocks and elements with custom visibility settings are listed at the bottom
of the editor sidebar.

You can preview how the page would look by clicking the :icon`fa-eye`
(:guilabel:`visible`) button to hide a block or element, or the :icon`fa-eye-slash
(:guilabel:`hidden`) to show it in the website editor. 
```
I prefer "editor sidebar" as it's more concise, but I'm not sure it's the official word :)

I'm not sure saving has any purpose here?

You could also include a screenshot since this part of the UI is quite hidden, so users know what to look for.

![2025-05-16_16-16](https://github.com/user-attachments/assets/3eca2b56-25f9-4358-982f-c95f06d5489c)